contentcuration/contentcuration/views/files.py

```diff
@@ -49,9 +51,13 @@ def upload_url(request):
 
     filepath = generate_object_storage_name(checksum, filename)
     checksum_base64 = codecs.encode(codecs.decode(checksum, "hex"), "base64").decode()
-    url = get_presigned_upload_url(filepath, checksum_base64, 600, content_length=size)
+    retval = get_presigned_upload_url(filepath, checksum_base64, 600, content_length=size)
+
+    # make sure we have both the mimetype and uploadURL values returned
+    assert isinstance(retval["mimetype"], str), "No mimetype present on get_presigned_upload_url return value!"
```

Comment:
These lines look like they belong in the test of this API, not in the live code.

Reply:
that's true, and we actually have tests for this. I just forgot we had when I was writing this!

Reply:
Removed those lines.